internal/pkg/service/stream/definition/repository/branch_repo.go

```diff
@@ -1,190 +0,0 @@
-package repository
-
```

Comment:
First commit: removed old Branch repository code, there is too much differences.